linux/jdk/redhat/src/main/packaging/microsoft/11/msopenjdk-11.spec

```diff
@@ -95,12 +93,12 @@ Provides: jre-11-%{java_provides}
 Provides: jre-%{java_provides}
 
 # First architecture (x64)
-Source0: %{source_url_base}/microsoft-jdk-%{upstream_version_url}-linux-%{vers_arch}.tar.gz
-Source1: %{source_url_base}/microsoft-jdk-%{upstream_version_url}-linux-%{vers_arch}.tar.gz.sha256sum.txt
+Source0: %{source_url_base}/microsoft-jdk-%{spec_version}-linux-%{vers_arch}.tar.gz
```

Comment:
Is that change in version string sourcing correct?

Reply:
Yes, I just double check it and it is working fine.